docs/rapidstart.md

```diff
@@ -0,0 +1,717 @@
+    --conf "spark.sql.extensions=io.delta.sql.DeltaSparkSessionExtension" \
+    --conf "spark.sql.catalog.spark_catalog=io.unitycatalog.spark.UCSingleCatalog" \
+    --conf "spark.hadoop.fs.s3.impl=org.apache.hadoop.fs.s3a.S3AFileSystem" \
+    --conf "spark.sql.catalog.unity=io.unitycatalog.spark.UCSingleCatalog" \
+    --conf "spark.sql.catalog.unity.uri=http://localhost:8080" \
+    --conf "spark.sql.catalog.unity.token=" \
+    --conf "spark.sql.defaultCatalog=unity"
+```
+
+#### Azure
+
+```
+bin/spark-sql --name "azure-uc-test" \
+    --master "local[*]" \
+    --packages "org.apache.hadoop:hadoop-azure:3.3.6,io.delta:delta-spark_2.12:3.2.1,io.unitycatalog:unitycatalog-spark_2.12:0.2.0" \
+    --conf "spark.sql.extensions=io.delta.sql.DeltaSparkSessionExtension" \
+    --conf "spark.sql.catalog.spark_catalog=io.unitycatalog.spark.UCSingleCatalog" \
+    --conf "spark.sql.catalog.unity=io.unitycatalog.spark.UCSingleCatalog" \
+    --conf "spark.sql.catalog.unity.uri=http://localhost:8080" \
+    --conf "spark.sql.catalog.unity.token=" \
+    --conf "spark.sql.defaultCatalog=unity"
+```
+
+#### GCS
+
+```
+bin/spark-sql --name "gcs-uc-test" \
+    --master "local[*]" \
+    --jars "https://repo1.maven.org/maven2/com/google/cloud/bigdataoss/gcs-connector/3.0.2/gcs-connector-3.0.2-shaded.jar" \
+    --packages "io.delta:delta-spark_2.12:3.2.1,io.unitycatalog:unitycatalog-spark_2.12:0.2.0" \
+    --conf "spark.sql.extensions=io.delta.sql.DeltaSparkSessionExtension" \
+    --conf "spark.sql.catalog.spark_catalog=io.unitycatalog.spark.UCSingleCatalog" \
+    --conf "spark.hadoop.fs.gs.impl=com.google.cloud.hadoop.fs.gcs.GoogleHadoopFileSystem" \
+    --conf "spark.hadoop.fs.AbstractFileSystem.gs.impl=com.google.cloud.hadoop.fs.gcs.GoogleHadoopFS" \
+    --conf "spark.hadoop.fs.gs.auth.type=ACCESS_TOKEN_PROVIDER" \
+    --conf "spark.hadoop.fs.gs.auth.access.token.provider=io.unitycatalog.spark.GcsVendedTokenProvider" \
+    --conf "spark.sql.catalog.unity=io.unitycatalog.spark.UCSingleCatalog" \
+    --conf "spark.sql.catalog.unity.uri=http://localhost:8080" \
+    --conf "spark.sql.catalog.unity.token=" \
+    --conf "spark.sql.defaultCatalog=unity"
+```
+
+### Quick Start UC from Spark SQL Shell
+
+Let’s start by running some quick commands from the Spark SQL shell.    The following SHOW SCHEMA will show the `default` schema that is by default included in the initial UC configuration. 
+
+```sql
+-- Show schemas
+SHOW SCHEMAS;
+```
+
+ 
+
+Next, let’s get the list of tables referenced within your local UC
+
+```sql
+-- Show tables
+SHOW TABLES IN default;
+```
+
+with the output similar to:
+
+```bash
+marksheet
+marksheet_uniform
+numbers
+user_countries
+Time taken: 0.21 seconds, Fetched 4 row(s)
+```
+
+Let’s query the first five rows of the `marksheet` table.
+
+```sql
+SELECT * FROM default.marksheet LIMIT 5;
```

Comment:
Does this work without specifying `catalog_name.schema_name.table_name`?

Reply:
Yes, on L78, the `bin/spark-sql` call denotes the catalog being used per ` --conf "spark.sql.defaultCatalog=unity"`